File: jscomp/test/array_data_util.ml

module A = Belt.Array

(* []*)
let range i j = 
  A.makeBy (j - i + 1) (fun k -> k + i )

let randomRange i j  =
    let v= (A.makeBy (j - i + 1) (fun k -> k + i )) in 
    A.shuffleInPlace v ; 
    v 
    
    

File: jscomp/test/array_data_util.js
'use strict';

var Belt_Array = require("../../lib/js/belt_Array.js");

function range(i, j) {
  return Belt_Array.makeBy((j - i | 0) + 1 | 0, (function (k) {
                return k + i | 0;
              }));
}

function randomRange(i, j) {
  var v = Belt_Array.makeBy((j - i | 0) + 1 | 0, (function (k) {
          return k + i | 0;
        }));
  Belt_Array.shuffleInPlace(v);
  return v;
}

var A = 0;

exports.A = A;
exports.range = range;
exports.randomRange = randomRange;
/* No side effect */
